### PR TITLE
Conditional constructions

### PR DESCRIPTION
**Pull request by AndreyShushket**

### CODE REVIEW

File: ConditionalConstructions/Program.cs [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Exercise 1");
+            
+            // First Task
+            Console.WriteLine("Task 1");
+            Console.WriteLine("Enter the number");
+            int firstTaskNumber = Convert.ToInt32(Console.ReadLine());
+
+            int evenOrNot = firstTaskNumber % 2;
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: enteredNumber

File: ConditionalConstructions/Program.cs [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Exercise 1");
+            
+            // First Task
+            Console.WriteLine("Task 1");
+            Console.WriteLine("Enter the number");
+            int firstTaskNumber = Convert.ToInt32(Console.ReadLine());
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: try catch

File: ConditionalConstructions/Program.cs [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+            Console.WriteLine("");
+
+            // Second Task
+            Console.WriteLine("Task 2");
+            Console.WriteLine("Enter the mounth number");
+
+            int MounthNumber = Convert.ToInt32(Console.ReadLine());
+
+            if (MounthNumber >= 1 && MounthNumber <= 2)
+            {
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs:43]: switch

File: ConditionalConstructions/Program.cs [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+            Console.WriteLine("Enter number a");
+            int thirdTaskNumA = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("Enter number b");
+            int thirdTaskNumB = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("Enter number c");
+            int thirdTaskNumC = Convert.ToInt32(Console.ReadLine());
+
+            if (thirdTaskNumA < thirdTaskNumB && thirdTaskNumB < thirdTaskNumC )
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: Console.WriteLine("Expression a < b < c is " + (thirdTaskNumA < thirdTaskNumB && thirdTaskNumB < thirdTaskNumC) );

File: ConditionalConstructions/Program.cs [hunk 5]
```diff
@@ -0,0 +1,390 @@
...
+            }
+            Console.WriteLine("");
+            
+            //Fourth Task
+            Console.WriteLine("Task 4");
+            Console.WriteLine("Enter string on russin");
+
+            string FourthTaskString = Console.ReadLine();
+
+            if(FourthTaskString.Contains("тест"))
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: 1 line

File: ConditionalConstructions/Program.cs [hunk 6]
```diff
@@ -0,0 +1,390 @@
...
+            {
+                Console.WriteLine("String do not contain тест");
+            }
+            Console.WriteLine("");
+            
+            //Fifth Task
+            Console.WriteLine("Task 5");
+
+            Console.WriteLine("Enter string Ru or En");
+            string firfhTaskString = Console.ReadLine();
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: renamed variable

File: ConditionalConstructions/Program.cs [hunk 7]
```diff
@@ -0,0 +1,390 @@
...
+            string enString = "qwertyuiopasdfghjklzxcvbnmMNBVCXZLKJHGFDSAPOIUYTREWQ";
+            string ruString = "йцукенгшщзхъфывапролджэячсмитьбюЙЦУКЕНГШЩЗХЪФЫВАПРОЛДЖЭЯЧСМИТЬБЮ";
+
+            if (enString.Contains(firfhTaskString[0]))
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: if( interedString.Equals("Ru"))
{
}
else if(==EN)
{
}
Else{}

File: ConditionalConstructions/Program.cs [hunk 8]
```diff
@@ -0,0 +1,390 @@
...
+            }
+            Console.WriteLine("");
+            
+            //Sixth Task
+
+            Console.WriteLine("Task 6");
+            Console.WriteLine("Enter number ");
+            int sixthTaskNumber = Convert.ToInt32(Console.ReadLine());
+
+            sixthTaskNumber = sixthTaskNumber % 8;
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: reduce variable

File: ConditionalConstructions/Program.cs [hunk 9]
```diff
@@ -0,0 +1,390 @@
...
+                    count++;
+                }
+                
+            }
+            if (count < 2)
+            {
+                Console.WriteLine("Less then two 7");
+            }
+            else if (count == 2)
+            {
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: if (count == 2)

File: ConditionalConstructions/Program.cs [hunk 10]
```diff
@@ -0,0 +1,390 @@
...
+            }
+            Console.WriteLine("");
+            
+            //Third Task
+            Console.WriteLine("Task 3");
+            Console.WriteLine("Enter xxxx number");
+
+            int thirdTaskNum = Convert.ToInt32(Console.ReadLine());
+            int firstHalh = 0;
+            int secHalf = 0;
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: secondHalf

File: ConditionalConstructions/Program.cs [hunk 11]
```diff
@@ -0,0 +1,390 @@
...
+            for (int i = 0; i < 2; i++)
```
Comment (VladimirVolkovVD) [inline ConditionalConstructions/Program.cs]: refactor

**Review by VladimirVolkovD — APPROVED** [review]

**Pull request merged**